test/taoensso/faraday/tests/requests.clj

```diff
@@ -6,92 +6,100 @@
    [taoensso.nippy   :as nippy])
 
   (:import
-   [com.amazonaws.services.dynamodbv2.model
-    AttributeAction
-    AttributeDefinition
-    AttributeValue
-    AttributeValueUpdate
-    BatchGetItemRequest
-    BatchGetItemResult
-    BatchWriteItemRequest
-    BatchWriteItemResult
-    Condition
-    ConsumedCapacity
-    ComparisonOperator
-    CreateGlobalSecondaryIndexAction
-    CreateTableRequest
-    CreateTableResult
-    DeleteItemRequest
-    DeleteItemResult
-    DeleteRequest
-    DeleteTableRequest
-    DeleteTableResult
-    DescribeTableRequest
-    DescribeTableResult
-    ExpectedAttributeValue
-    GetItemRequest
-    GetItemResult
-    ItemCollectionMetrics
-    KeysAndAttributes
-    KeySchemaElement
-    KeyType
-    ListTablesRequest
-    ListTablesResult
-    LocalSecondaryIndex
-    LocalSecondaryIndexDescription
-    GlobalSecondaryIndex
-    GlobalSecondaryIndexDescription
-    GlobalSecondaryIndexUpdate
-    Projection
-    ProjectionType
-    ProvisionedThroughput
-    ProvisionedThroughputDescription
-    PutItemRequest
-    PutItemResult
-    PutRequest
-    QueryRequest
-    QueryResult
-    ReturnValue
-    ScanRequest
-    ScanResult
-    Select
-    TableDescription
-    UpdateItemRequest
-    UpdateItemResult
-    UpdateTableRequest
-    UpdateTableResult
-    WriteRequest
-
-    ConditionalCheckFailedException
-    DeleteGlobalSecondaryIndexAction
-    InternalServerErrorException
-    ItemCollectionSizeLimitExceededException
-    LimitExceededException
-    ProvisionedThroughputExceededException
-    ResourceInUseException
-    ResourceNotFoundException
-    UpdateGlobalSecondaryIndexAction]))
+    [com.amazonaws.services.dynamodbv2.model
```

Comment:
May I request that the indenting not be changed here? Just creating some unnecessary diff noise. Thanks!